code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -212,7 +217,11 @@
 
 /obj/machinery/airalarm/Initialize(mapload)
 	. = ..()
+	if(!mapload)
+		return
```

Comment:
What's this block for? Why should this not run the other procs if instanced in regular play?